packages/liveblocks-core/src/__tests__/_utils.ts

```diff
@@ -32,12 +32,12 @@ import { remove } from "../utils";
 import type { JsonStorageUpdate } from "./_updatesUtils";
 import { serializeUpdateToJson } from "./_updatesUtils";
 
-function makeRoomToken(actor: number): RoomAuthToken {
+function makeRoomToken(actor: number, scopes: string[] = []): RoomAuthToken {
```

Comment:
If we can, I'd prefer to make this argument explicit and not use a default value. It might make the call sites more explicit.

```suggestion
function makeRoomToken(actor: number, scopes: string[]): RoomAuthToken {
```

Reply:
Sounds good, would it be OK with explicit on just the function `makeRoomToken`?